server/controller/src/main/resources/db/migration/V00003__Add_view_config.sql

```diff
@@ -0,0 +1,15 @@
+-- ----------------------------
```

Comment:
Incremental sql scripts are used between releases. In scope of one release, we should add the script to the only one script file that belongs to this release